### PR TITLE
Add Steven Foltz as author

### PR DESCRIPTION
**Pull request by sjspielman**
Steven Foltz is added an an author with contribution of "Validation."

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### CODE REVIEW

**Comment by sjspielman** [comment]
@envest, does this look ok to you? Have a different preferred email?

File: content/metadata.yaml [hunk 1]
```diff
@@ -444,6 +444,16 @@ authors:
     affiliations:
       - Hackensack Meridian School of Medicine
       - Hackensack University Medical Center
+  - github: envest
+    name: Steven Foltz
```
Comment (envest) [inline content/metadata.yaml:448]:
```suggestion
    name: Steven M. Foltz
```

File: content/metadata.yaml [hunk 2]
```diff
@@ -444,6 +444,16 @@ authors:
...
+    initials: SF
```
Comment (envest) [inline content/metadata.yaml:449]:
```suggestion
    initials: SMF
```

**Review by envest — APPROVED** [review]
Thank you! Added my middle initial

**Pull request merged**